Template.php

```diff
@@ -444,6 +444,19 @@ public function add_if_new($param_name, $value) {
   }
 
   protected function get_identifiers_from_url() {
+    if ($this->blank('url') && !$this->blank('website')) {  // No URL, but a website
+      $url = trim($this->get('website'));
+      if (strtolower(substr( $url, 0, 4 )) !== "http" ) {
+        $url = "https://" . $url; // Try it with http
```

Comment:
Could we go a step further with a regexp to catch typos such as `website = ttp:/web.com`?  (Might not be necessary in enough cases to justify)
Also is it worth testing `http` as well as `https` to catch sites that lack SSL?

Reply:
Improving